saas/saas.go

```diff
@@ -26,7 +26,9 @@ import (
 )
 
 // Writer writes results to SaaS
-type Writer struct{}
+type Writer struct {
+	TimeoutSec int
```

Comment:
Looking at other writers, they often have implementations that include Cnf and Proxy.
```suggestion
	Cnf        config.SaasConf
	Proxy      string
	TimeoutSec int
```
https://github.com/future-architect/vuls/blob/b4bd2ac8b914f83aa6bb3e5ce6bca4566f0342f7/reporter/slack.go#L24-L25